apps/fortuna/src/state.rs

```diff
@@ -42,7 +43,7 @@ impl PebbleHashChain {
         }
     }
 
-    pub fn from_config(
+    pub async fn from_config(
```

Comment:
can i suggest making a separate constructor method that spawns the original from_config in a blocking thread? like from_config_async